components/views/auth/sign-in.tsx

```diff
@@ -1,32 +1,73 @@
-"use client"
+"use client";
 
 import { Button } from "@/components/ui/button";
-
+import Image from "next/image";
+import { PiAsteriskSimpleBold } from "react-icons/pi";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card"
 import { useSession, signIn, signOut } from "next-auth/react";
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
 
 const SignInView = () => {
-
   const { data: session } = useSession();
```

Comment:
- I suggest if we build such logic in AuthContext class.
- AuthContext should be responsible to identify user-type and redirect the correct endpoint